clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -683,21 +686,55 @@ public Node provide() {
         }
     }
 
+    class RetryContext {
+
+        private int tries = 0;
+        private long nextAllowedTryMs = 0;
+
+        public int getTries() {
+            return tries;
+        }
+
+        public long getNextAllowedTryMs() {
+            return nextAllowedTryMs;
+        }
+
+        private void updateTries(int currentTries) {
+            this.tries = currentTries + 1;
+        }
+
+        private void updateNextAllowTryMs() {
+            double jitter = Math.random() * 0.4 + 0.8;
```

Comment:
Instead of hardcoding these numbers, can we define them explicitly to make the code easier to read? Like this for example:

```java
double JITTER_MIN_VALUE = 0.8;
double JITTER_MAX_VALUE = 1.2;

double jitter = Math.random() * (JTTER_MAX_VALUE - JITTER_MIN_VALUE) + JITTER_MIN_VALUE;
```

Reply:
Yes, adding these two final double variables to the CallRetryContext class.